libfwupdplugin/fu-usb-bos-descriptor.c

```diff
@@ -35,6 +37,28 @@ G_DEFINE_TYPE_EXTENDED(FuUsbBosDescriptor,
 		       G_IMPLEMENT_INTERFACE(FWUPD_TYPE_CODEC,
 					     fu_usb_bos_descriptor_codec_iface_init));
 
+static void
+fu_usb_bos_descriptor_export(FuFirmware *firmware, FuFirmwareExportFlags flags, XbBuilderNode *bn)
+{
+	FuUsbBosDescriptor *self = FU_USB_BOS_DESCRIPTOR(firmware);
+	fu_xmlb_builder_insert_kx(bn, "dev_capability_type", self->bos_cap.bDevCapabilityType);
+}
+
+static gboolean
+fu_usb_bos_descriptor_build(FuFirmware *firmware, XbNode *n, GError **error)
+{
+	FuUsbBosDescriptor *self = FU_USB_BOS_DESCRIPTOR(firmware);
+	guint64 tmp;
+
+	/* simple properties */
+	tmp = xb_node_query_text_as_uint(n, "dev_capability_type", NULL);
```

Comment:
why not pass error here?

Reply:
I did the last thing, thanks for the reminder.